internal/api/v1/services/instance.go

```diff
@@ -28,8 +28,8 @@ func NewInstanceService(repo *repos.InstanceRepository, jobService *Job) *Instan
 }
 
 // ListInstances retrieves a paginated list of instances
-func (s *Instance) ListInstances(ctx context.Context, opts *models.ListOptions) ([]models.Instance, error) {
-	return s.repo.List(ctx, opts)
+func (s *Instance) ListInstances(ctx context.Context, ownerID uint, opts *models.ListOptions) ([]models.Instance, error) {
+	return s.repo.List(ctx, ownerID, opts)
```

Comment:
If there is an exception for `models.AdminID` it should have been processed here in the services or in the middleware layer.